src/test/java/com/sprint/mission/discodeit/repository/MessageRepositoryTest.java

```diff
@@ -0,0 +1,122 @@
+package com.sprint.mission.discodeit.repository;
+
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.sprint.mission.discodeit.config.JpaConfig;
+import com.sprint.mission.discodeit.entity.Channel;
+import com.sprint.mission.discodeit.entity.ChannelType;
+import com.sprint.mission.discodeit.entity.Message;
+import java.util.List;
+import java.util.UUID;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.context.annotation.Import;
+import org.springframework.data.domain.Sort;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
+
+@Import(JpaConfig.class)
+@ExtendWith(SpringExtension.class) // Junit5 + Spring
+@DataJpaTest
+public class MessageRepositoryTest {
```

Comment:
```suggestion
@DataJpaTest
@Import(JpaConfig.class)
public class MessageRepositoryTest {
```

DataJpaTest 어노테이션은 이미 ExtendWith(SpringExtension.class)을 포함하고 있습니다.
추가로, 아래 어노테이션에 대해 확인해보세요. 아마 테스트를 위한 h2 디비가 따로 만들어질거에요.

https://emgc.tistory.com/143
https://docs.spring.io/spring-boot/api/java/org/springframework/boot/test/autoconfigure/jdbc/AutoConfigureTestDatabase.html